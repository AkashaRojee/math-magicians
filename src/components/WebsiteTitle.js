const WebsiteTitle = () => (
  <span>Math Magicians</span>
);

export default WebsiteTitle;
