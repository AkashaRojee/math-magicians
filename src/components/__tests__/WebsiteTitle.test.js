import renderer from 'react-test-renderer';
import WebsiteTitle from '../WebsiteTitle';

test('WebsiteTitle renders correctly', () => {
  const tree = renderer
    .create(<WebsiteTitle />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});