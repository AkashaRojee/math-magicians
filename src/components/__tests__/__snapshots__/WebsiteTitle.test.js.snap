// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebsiteTitle renders correctly 1`] = `
<h1>
  Math Magicians
</h1>
`;
